recipes/flake8-junit-report/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "flake8-junit-report" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f8890c1ebe0acb516fefacddec4b802bca9f89bb07db933e4ee3cd11ceaa1e8b
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  entry_points:
+    - flake8_junit = junit_conversor.__main__:main
+
+requirements:
+  host:
+    - python >=2.7
+    - pip
+  run:
+    - python >=2.7
+    - flake8
+
+test:
+  requires:
+    - flake8
+  imports:
+   - junit_conversor
+  commands:
+    - flake8 test_flake8_file.py --output-file flake8.txt || echo "expected to set exit code 1"
```

Comment:
To use `test_flake8_file.py` in the tests, you have to define it in `files`. See https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-files